swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = None, condition = 0):
```

Comment:
Category can be set to `""` as a default. Strings are immutable in Python, so you don't need to use None. None is for when something is mutable (like a list or a dictionary).